<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>IronTrans</string>
    </property>
   </widget>
   <widget class="QWidget" name="main_widget" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>541</width>
      <height>361</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>221</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="journalButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Журнал</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="userList_button">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Список пользователей</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="proList_button">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Список работников</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_journal" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>671</width>
      <height>541</height>
     </rect>
    </property>
    <widget class="QTableWidget" name="journalTable">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>511</width>
       <height>431</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>Дата</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Время</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Событие</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Пользователь</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="backFromJournal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_user" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>701</width>
      <height>521</height>
     </rect>
    </property>
    <widget class="QTableWidget" name="userTable">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>261</width>
       <height>431</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>Пользователь</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Пароль</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="backFromUser">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_pro" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>761</width>
      <height>541</height>
     </rect>
    </property>
    <widget class="QPushButton" name="backFromPro">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="proTable">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>421</width>
       <height>441</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Пользователь</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Пароль</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Должность</string>
      </property>
     </column>
    </widget>
   </widget>
   <zorder>label_11</zorder>
   <zorder>widget_journal</zorder>
   <zorder>widget_user</zorder>
   <zorder>widget_pro</zorder>
   <zorder>main_widget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
